pkg/types/backend.go

```diff
@@ -265,6 +248,27 @@ func (s *Stub) UnmarshalConfig() (*StubConfigV1, error) {
 	return config, nil
 }
 
+func (s *Stub) SanitizeConfig() error {
+	var config StubConfigV1
+	err := json.Unmarshal([]byte(s.Config), &config)
+	if err != nil {
+		return err
+	}
+
+	// Remove secret values from config
+	for i := range config.Secrets {
+		config.Secrets[i].Value = ""
```

Comment:
Should this be some placeholder value that indicates it exists despite not being shown here?

Reply:
Yeah, could be helpful to the person seeing it in the browser. Something like `"*****"`? Did you have something in mind?

Reply:
Thought about this a little more... I think I'll leave it empty, at least for now. This ends up being in both the deployment and the task API responses and I wouldn't want someone to do something like below and think there's actually a useful value there. 

```python
# someone might do this
if secret["value"]:
    # do something

# when they should do this
if secret["value"] == "*****":
    # do something
```